src/ojo/agent/OjoUtils.cc

```diff
@@ -352,7 +352,11 @@ void appendToJson(const std::string fileName,
 {
   Json::Value result;
   Json::FastWriter jsonBuilder;
+#if JSONCPP_VERSION_MAJOR > 0 && JSONCPP_VERSION_MINOR > 6
```

Comment:
Don't we need an alternative function then for older versions? Otherwise we can get different results with older versions.

Reply:
Nice idea. Thanks.

Reply:
I have made the required changes to keep the output same on all versions. Can you please review it again?